nix/overlays/required.nix

```diff
@@ -7,11 +7,14 @@ let
   addRealTimeTestLogs = drv: overrideCabal drv (attrs: {
     testTarget = "--show-details=streaming";
   });
+  -- We do this for things where we need to run the plugin, but where Haddock chokes on it. We can't just turn it off,
```

Comment:
nix uses `#` for comments, this breaks CI

Reply:
:man_facepalming: This does actually work, I just added the comment afterwards.